remote-trait-object/src/packet.rs

```diff
@@ -200,9 +190,9 @@ impl Packet {
         packet
     }
 
-    pub fn new_request(service_name: String, method: MethodId, args: &[u8]) -> Self {
+    pub fn new_request(object: ServiceObjectId, method: MethodId, args: &[u8]) -> Self {
```

Comment:
`object` is too general. How about `object_id` or `service_object_id`?

```suggestion
    pub fn new_request(service_object_id: ServiceObjectId, method: MethodId, args: &[u8]) -> Self {
```